datastore/admin-api/DatastoreAdmin.Samples.Tests/GetIndexTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Linq;
+using Xunit;
+
+[Collection(nameof(DatastoreAdminFixture))]
+public class GetIndexTest
+{
+    private readonly DatastoreAdminFixture _datastoreAdminFixture;
+
+    public GetIndexTest(DatastoreAdminFixture datastoreAdminFixture)
+    {
+        _datastoreAdminFixture = datastoreAdminFixture;
+    }
+
+    [Fact]
+    public void TestGetIndex()
+    {
+        // Currently, we don't have any API to create index.
+        // It Just verify that if any index is available than GetIndex does not throw any Exception.
+        ListIndexesSample listIndexesSample = new ListIndexesSample();
+        GetIndexSample getIndexSample = new GetIndexSample();
+        var indexes = listIndexesSample.ListIndexes(_datastoreAdminFixture.ProjectId).ToList();
+        if (indexes.Any())
```

Comment:
Also, add an else to this if, that calls the sample with any indexId, and verifies that the exception thrown is NotFound.

Reply:
Done.